### DISCUSSION TITLE
freerdp-nightly - fedora

### DISCUSSION THREAD

**Discussion by aebenjam** [FreeRDP/FreeRDP#9667] [General]
Noticed that the current yum.repos.d config for the [fedora nightly build](https://ci.freerdp.com/job/freerdp-nightly-rpm-mock/) is missing a "fedora-" in the baseurl right before $releasever.  Here's the correct path, as far as I can see:

baseurl=https://pub.freerdp.com/repositories/rpm/fedora/fedora-$releasever/freerdp-nightly/

Adam

**Comment by akallabeth** [comment]
@aebenjam thank you, yes, the page required an update.
